master/getting-started/kubernetes/upgrade/upgrade.md

```diff
@@ -30,10 +30,17 @@ Once you have upgraded {{site.prodname}}, you can [complete the upgrade](#comple
 1. Check the status of the upgrade as follows.
 
    ```
-   kubectl rollout status ds/calico-node -n kube-system
+   kubectl get pods -n kube-system | grep calico-node
```

Comment:
or `watch kubectl get pods -n kube-system | grep calico-node` perhaps?

Reply:
Or actually, `watch kubectl get pods -n kube-system`

Reply:
"Wait until each pod has the STATUS of Running."

Reply:
I might suggest something more like this for the `1/2` part and you could maybe even put it below the columnar table.

```
> **Note**: The {{site.noderunning}} pods will report `1/2` in the `Ready` column, as shown. 
{: .alert .alert-info}
```